modules/core/04-channel/keeper/packet_test.go

```diff
@@ -351,30 +351,6 @@ func (suite *KeeperTestSuite) TestRecvPacket() {
 				channel := path.EndpointB.GetChannel()
 				channel.State = types.FLUSHING
 				path.EndpointB.SetChannel(channel)
-
```

Comment:
i removed these tests bc we aren't checking these sequences in `RecvPacket` anymore s